admin_server/routes/auth.js

```diff
@@ -0,0 +1,146 @@
+const router = require('express').Router();
+const axios = require('axios');
+const jwt = require('jsonwebtoken');
+
+const MSG91_URL = "https://control.msg91.com/api/";
```

Comment:
Can we have this MSG91_URL as an ENV ... will be easy if we had to change it tomorrow.

Reply:
if we change service, endpoint will not be the same. even if we create a wrapper function the params will not be the same.

Reply:
I mentioned the URL string.